lib/src/module.dart

```diff
@@ -498,6 +500,7 @@ abstract class Module {
 /**
  * Generated by ROHD - www.github.com/intel/rohd
  * Generation time: ${DateTime.now()}
+ * ROHD Version: ${Config.version}
```

Comment:
Nice, let's do the same thing in `wave_dumper.dart`.  Something like:
```
\$version
  ROHD v${Config.version}
\$end
```
In the `_writeHeader` function